packages/blade/private/shell/utils/providers.ts

```diff
@@ -102,14 +102,12 @@ export const transformToVercelBuildOutput = async (): Promise<void> => {
  * Transform to match the Cloudflare output structure.
  */
 export const transformToCloudflareOutput = async (): Promise<void> => {
-  const spinner = logSpinner(
-    'Transforming to output for Cloudflare (production)',
-  ).start();
+  const spinner = logSpinner('Transforming output for Cloudflare').start();
 
   const promises = new Array<Promise<unknown>>(
     writeFile(
       path.join(outputDirectory, '.assetsignore'),
-      ['edge-worker.js', 'edge-worker.js.map', '_routes.json'].join('\n'),
+      ['edge-worker.js', 'edge-worker.js.map', '_routes.json', '*.map'].join('\n'),
```

Comment:
Couldn't we remove `'edge-worker.js.map'` here since we're already ignoring all `.map` files?